examples/teleop-elm/package.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "@viamrobotics/sdk-teleop-demo",
-  "version": "0.0.1",
-  "description": "Viam SDK tele-operation demo in Elm",
+  "name": "example",
+  "description": "An example tele-operation application using @viamrobotics/sdk",
   "scripts": {
+    "preinstall": "cd ../.. && make build",
```

I like using the `-C` option for this rather than manually `cd`'ing

```suggestion
    "preinstall": "make -C ../.. build",
```